code/notificationbuilder/src/test/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/ZeroBezelNotificationBuilderTest.kt

```diff
@@ -0,0 +1,234 @@
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.notificationbuilder.internal.builders
+
+import android.app.Activity
+import android.app.Notification
+import android.content.Context
+import android.graphics.Bitmap
+import android.os.Bundle
+import android.view.View
+import android.widget.RemoteViews
+import com.adobe.marketing.mobile.notificationbuilder.PushTemplateConstants
+import com.adobe.marketing.mobile.notificationbuilder.internal.PushTemplateImageUtils
+import com.adobe.marketing.mobile.notificationbuilder.internal.PushTemplateImageUtils.cacheImages
+import com.adobe.marketing.mobile.notificationbuilder.internal.PushTemplateImageUtils.getCachedImage
+import com.adobe.marketing.mobile.notificationbuilder.internal.templates.MockAEPPushTemplateDataProvider
+import com.adobe.marketing.mobile.notificationbuilder.internal.templates.ZeroBezelPushTemplate
+import com.adobe.marketing.mobile.notificationbuilder.internal.templates.replaceValueInMap
+import com.adobe.marketing.mobile.notificationbuilder.internal.util.IntentData
+import com.adobe.marketing.mobile.notificationbuilder.internal.util.MapData
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.mockkConstructor
+import io.mockk.mockkObject
+import io.mockk.verify
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertNotNull
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.Shadows
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [31])
+class ZeroBezelNotificationBuilderTest {
+
+    private lateinit var context: Context
+    private lateinit var dataMap: MutableMap<String, String>
+    private lateinit var mockBitmap: Bitmap
+    private lateinit var trackerActivityClass: Class<out Activity>
+    private lateinit var mockBundle: Bundle
+
+    @Before
+    fun setup() {
+        context = RuntimeEnvironment.getApplication()
+        dataMap = MockAEPPushTemplateDataProvider.getMockedAEPDataMapWithAllKeys()
+        mockkObject(PushTemplateImageUtils)
+        mockkConstructor(RemoteViews::class)
+        mockBitmap = mockk<Bitmap>()
+        trackerActivityClass = DummyActivity::class.java
+        mockBundle = MockAEPPushTemplateDataProvider.getMockedAEPBundleWithAllKeys()
+        every { getCachedImage(any()) } answers { mockBitmap }
+    }
+
+    @Test
+    fun `verify construct with image downloaded and collapsedStyle is img`() {
+        val pushTemplate = ZeroBezelPushTemplate(MapData(dataMap))
+        every { cacheImages(any()) } answers { 2 }
```

Comment:
nit: why is `answers { 2 }`?Shouldn't it be `answers { 1 }`

Reply:
Picked a random number here will update